core-bundle/src/Twig/Runtime/SanitizerRuntime.php

```diff
@@ -18,6 +18,7 @@
 use Twig\Environment;
 use Twig\Extension\RuntimeExtensionInterface;
 
+// Backwards compatibility: To be removed in Contao 6
```

Comment:
Would it be better to deprecate the class?

```suggestion
/**
 * @deprecated Deprecated since Contao 5.7, to be removed in Contao 6
 */
```

Reply:
I don't think a deprecation is correct. This class isn't used directly, the adjustments in there are needed for the current setup though. In the future without input encoding we need to remove it (otherwise it would be unsafe) and things will automatically work. So this is merely a reminder for us to not forget it. 🙂